### ISSUE TITLE
Error: "can't build with source format '3.0 (native)': native package version may not have a revision"

### ISSUE THREAD

**Issue by netfall-tv** [jamesdbloom/grunt-debian-package#23]
On Debian Jessie this task fails with the following error message, which is only displayed when --verbose has been added to the grunt command:

> can't build with source format '3.0 (native)': native package version may not have a revision

Without --verbose a generic and not helpful error message is displayed that may lead into the wrong direction. On a recent Ubuntu this error doesn't happen. (But probably will on future releases.)

The reason for this is that the package is built as **native** package whereas the version generated by this grunt task indicates that it is a **non-native** package because it uses a hyphen (like _1.2.3-4_ instead of just _1.2.3_).

My workaround for this is to simply replace the hyphen with a dot. In 
`node_modules/grunt-debian-package/packaging/debian/changelog`
replace
`${name} (${version}-${build_number}) experimental; urgency=low`
with
`${name} (${version}.${build_number}) experimental; urgency=low`
Now packaging should work as expected, however the version number will now show up as _1.2.3.4_

This is also probably the correct fix for this problem since _revision_ and _build number_ are two different things anyway.

**Comment by ddevaal** [comment]
Good catch. I have the same problem. Thanks @Netfall for the temporary fix.
